charts/invenio/templates/_helpers.tpl

```diff
@@ -50,6 +50,27 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 
+{{/*
+Return the proper Invenio image name
+*/}}
+{{- define "invenio.image" -}}
+{{- $registryName := .Values.image.registry -}}
+{{- $repositoryName := .Values.image.repository -}}
+{{- $separator := ":" -}}
+{{- $termination := .Values.image.tag | default .Chart.AppVersion | toString -}}
+
+{{- if .Values.image.digest }}
+  {{- $separator = "@" -}}
+  {{- $termination = .Values.image.digest | toString -}}
+{{- end -}}
+
+{{- if $registryName }}
```

Comment:
Why cover for when `image.registry` is unset? 🤔 Is that just for people using Docker Hub? If it's just that, then I think we should make them be explicit with setting the registry name in favor of simpler code at our end.